From 204ef1f1054b94b4da5621ea16f5ea839b97f378 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Tue, 14 Aug 2018 14:48:43 +0300
Subject: [PATCH] btl: move BTLID to scratchpad 31

We will always need the BTLID, and we don't need the U-Boot size.
Move it to scratchpad 31 (overwrite U-Boot size).

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/board/habanalabs/common/btl/primary.S b/board/habanalabs/common/btl/primary.S
index f098c2a..66d55aa 100644
--- a/board/habanalabs/common/btl/primary.S
+++ b/board/habanalabs/common/btl/primary.S
@@ -44,11 +44,10 @@
  * passing flash layout to CPU SW
  * ------------------------------
  */
-cfg mmPSOC_GLOBAL_CONF_SCRATCHPAD_27, BTLID
 cfg mmPSOC_GLOBAL_CONF_SCRATCHPAD_28, SPL_OFFSET
 cfg mmPSOC_GLOBAL_CONF_SCRATCHPAD_29, SPL_SIZE
 cfg mmPSOC_GLOBAL_CONF_SCRATCHPAD_30, UBOOT_OFFSET
-cfg mmPSOC_GLOBAL_CONF_SCRATCHPAD_31, UBOOT_SIZE
+cfg mmPSOC_GLOBAL_CONF_SCRATCHPAD_31, BTLID
 
 /* Set CPU0 Reset address to PREBOOT */
 cfg mmCPU_CA53_CFG_RST_ADDR_LSB_0, PREBOOT_ADDR_L
